frontend/src/app/app-body/app-body.tsx

```diff
@@ -14,45 +12,21 @@ interface IProps {
 @inject('testStore')
 @observer
 export class AppBody extends React.Component<IProps> {
-  @action
-  private toggleColour = () => {
-    this.props.testStore.fakeServerCall();
-  };
-
   public render() {
     return (
-      <Row gutter={16}>
-        <Col span={8}>
-          <div
-            style={{ color: this.props.testStore.colourActive ? 'red' : '' }}
-          >
-            Toggle the colour of this text NEW!
+      <div style={{minWidth: '1366px', display: 'flex'}}>
```

Comment:
read up on `flexbox` and `css grid` for more information on styling. This may not be the best solution!